app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -0,0 +1,37 @@
+class FeedbackReporter
+
+  attr_accessor :title, 
+    :description, 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.